src/titiler/extensions/titiler/extensions/templates/cog_viewer.html

```diff
@@ -955,8 +955,7 @@
     params.append('rescale', `${document.getElementById("data-min").value},${document.getElementById("data-max").value}`);
   }
 
-  const path_name = `${window.location.pathname}`.replace("viewer", "map");
-  console.log("path_name: ", path_name);
+  const path_name = `${window.location.pathname}`.replace("viewer", "WebMercatorQuad/map");
```

Comment:
😓 the only issue is that the `/map` endpoint might not be available

Reply:
yes

Reply:
I'll figure out how to make it conditionally visible.

Reply:
@hrodmn because the viewer is an extension, when we create the endpoint we have access to the factory attributes so in https://github.com/developmentseed/titiler/blob/4327ab83daeed7a17ba91d0f761d13c76c3a301d/src/titiler/extensions/titiler/extensions/viewer.py#L24-L42 you could pass a boolean in the template context